test/drivers/event_class/event_class.cpp

```diff
@@ -96,7 +96,7 @@ event_test::~event_test()
 	clear_ring_buffers();
 
 	/* 2 - clean all interesting syscalls. */
-	scap_clear_ppm_sc_mask(s_scap_handle);
+	clear_ppm_sc_mask();
```

Comment:
This whole commit looks like fixing a build error. Can we fold these changes into the commit(s) that made these fail instead?

Reply:
Nope, it is not ;) it's that `scap_clear_ppm_sc_mask` is no more an API; so i added the `clear_ppm_sc_mask` utility funcion in tests.

Reply:
Uh i see! I will fix it!

Reply:
Done!